Recent gcc -E splits output into multiple lines, separated by comments
--- dosemu-1.4.0.8/configure.ac.orig	2026-05-21 20:47:47.395180611 +0200
+++ dosemu-1.4.0.8/configure.ac	2026-05-21 21:02:06.083862029 +0200
@@ -645,7 +645,7 @@ if test "$GCC" = "yes" ; then
   fi
 
   AC_MSG_CHECKING(for glibc...)
-  set `printf '%b\n' '#include <features.h>\nXXAaZZ __GLIBC__ XXBbZZ __GLIBC_MINOR__'|${CC-cc} -E -|awk '/XXAaZZ/ {print $2 " " $4}'`
+  set `printf '%b\n' '#include <features.h>\nXXAaZZ __GLIBC__ XXBbZZ __GLIBC_MINOR__'|${CC-cc} -E -|grep -v '^#'|tr -d '\n'|awk '/XXAaZZ/ {print $2 " " $4}'`
   major=$1; minor=$2
   if test "$major" = "__GLIBC__"; then
     AC_MSG_ERROR([Sorry, you need glibc-2.1.3 or newer.])
